src/ceph-disk/tests/test_main.py

```diff
@@ -310,6 +311,10 @@ def get_partition_type(dev):
             assert expect == main.list_devices()
 
     def test_list_dmcrypt_data(self):
+        if platform.system() == "FreeBSD":
```

Comment:
@tchaikov @dachary 
FreeBSD does not have any of the dm stuff at all. The short cut was to disable here.
As I did at a few other places